roles/st2mistral/tasks/main.yml

```diff
@@ -1,18 +1,27 @@
 ---
+- name: Install st2mistral dependency
+  become: yes
+  package:
+    name: st2python
```

Comment:
In the bootstrap scripts we install `st2` and its dependencies before `st2mistral`, that is why it was never caught: https://github.com/StackStorm/st2-packages/blob/master/scripts/st2bootstrap-el6.sh#L653. However, we may not have another use cases besides this and docker, where anyone installs `st2mistral` before `st2`.